python/ray/data/_internal/execution/operators/output_splitter.py

```diff
@@ -91,6 +91,9 @@ def has_next(self) -> bool:
     def _get_next_inner(self) -> RefBundle:
         output = self._output_queue.popleft()
         self._metrics.on_output_dequeued(output)
+        if self._estimated_num_output_bundles is None:
+            self._estimated_num_output_bundles = 0
+        self._estimated_num_output_bundles += 1
```

Comment:
I think it's better to just inherit `num_outputs_total` from the input op. 
because output splitter doesn't change the number of blocks.